src/Pos/Update/Download.hs

```diff
@@ -33,34 +35,50 @@ import           Pos.WorkMode            (WorkMode)
 showHash :: Hash a -> FilePath
 showHash = toString . B16.encode . BA.convert
 
+-- TODO: if we're downloading update not for `cardano-sl`,
+-- but e. g. for `daedalus`, how do we check that version is new?
```

Comment:
I think that for now this should work correctly - we probably won't have updates not for `cardano-sl` for a while (if we will have them at all). Probably the good solution would be to create an issue about that and put it into backlog.